map/templates/map/nav-bar.html

```diff
@@ -2,8 +2,8 @@
 <nav class="navbar navbar-light bg-light" style="margin-bottom: 5px;">
     <a href="/" class="navbar-brand">Georide Road Trip</a>
     <span>
-        <a class="btn btn-outline-success my-2 my-sm-0" href="/modifie" role="button">Modifie Account</a>
-        <a class="btn btn-outline-danger my-2 my-sm-0" href="/disconect" role="button">Sign Out</a>
+        <a class="btn btn-outline-success my-2 my-sm-0" href="/modify" role="button">Modify Account</a>
+        <a class="btn btn-outline-danger my-2 my-sm-0" href="/disconect" role="button">Disconect</a>
```

Comment:
You change Sign out to disconnect

Reply:
My bad, I miss-checked my conflicts thinking there was only my updates.